Dockerfile

```diff
@@ -65,23 +65,11 @@ RUN set -ex \
     && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 \
     && useradd -ms /bin/bash -d ${AIRFLOW_HOME} airflow \
     && pip install -U pip setuptools wheel \
-    && pip install Cython \
-    && pip install pytz \
-    && pip install pyOpenSSL \
-    && pip install pandas==0.23.4 \
-    && pip install kubernetes==7.0.0 \
-    && pip install ndg-httpsclient \
-    && pip install pyasn1 \
-    && pip install marshmallow-sqlalchemy==0.17.0 \
-    && pip install apache-airflow[crypto,celery,postgres,hive,jdbc,mysql,gcp_api]==$AIRFLOW_VERSION \
-    && pip install redis==3.3.11 \
-    && pip install psycopg2 \
-    && pip install psycopg2-binary \
-    && pip install 'celery[redis]>=4.1.1,<4.2.0' \
-    && pip install 'tornado<6.0.0' \
-    && pip install wtforms==2.2.1 \
-    && pip install attrs==19.3.0 \
-    && apt-get purge --auto-remove -yqq $buildDeps \
+    && pip install --no-cache-dir -r requirements.txt \
```

Comment:
When building it says "ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'", I am not sure why, for example here the same command works:
http://github.com/medicode/docker-airflow/blob/81cbe537eab696c7cae797456604c68e8b05c030/airflow_test/Dockerfile#L4-L4

Reply:
Ah, you need to copy the requirements file, into the docker image first https://github.com/medicode/docker-airflow/blob/81cbe537eab696c7cae797456604c68e8b05c030/airflow_test/Dockerfile#L3